File: JavaScript/tripleSequence.js
function tripleSequence(base, length) {
  if (length === 0) {
    return [];
  }
  let arr = [base];
  while (arr.length < length) {
    let last = arr[arr.length - 1];
    let next = last * 3;
    arr.push(next);
  }
  return arr;
}

console.log(tripleSequence(2, 4));

File: Python/triple-sequence.py
def triple_sequence(base,length):
    if length == 0:
        return []
    arr = [base]
    while len(arr) < length:
        last = arr[len(arr) - 1]
        next = last * 3
        arr.append(next)

    return arr

print(triple_sequence(2,4))
